packages/template-retail-react-app/app/ssr.js

```diff
@@ -33,19 +33,54 @@ const options = {
 const runtime = getRuntime()
 
 const {handler} = runtime.createHandler(options, (app) => {
+    const getCSP = (nodeEnv) => {
+        const trustedMap = {
+            development: [
+                'localhost:*',
+                '*.commercecloud.salesforce.com',
+                '*.demandware.net',
+                '*.mobify-staging.com',
+                '*.mobify-storefront-staging.com',
+                '*.mobify-storefront.com',
+                'runtime.commercecloud.com'
+            ],
+            staging: [
+                '*.demandware.net',
+                '*.mobify-staging.com',
+                '*.mobify-storefront-staging.com',
+                '*.mobify-storefront.com',
+                '*.commercecloud.salesforce.com',
+                'runtime.commercecloud.com'
+            ],
+            production: [
+                '*.demandware.com',
+                '*.mobify.com',
+                '*.mobify-storefront.com',
+                '*.commercecloud.salesforce.com',
+                'runtime.commercecloud.com',
+                // TODO: Revert once we have a Runtime Admin we can use that's not in staging
+                '*.mobify-storefront-staging.com'
```

Comment:
Why do we have this here in the production CSP? It also has a TODO that implies this is for staging?

Reply:
This is a limitation of our Runtime Admin preview environment being deployed to cloud production, but loading data from cloud staging. However, this method of defining the CSP is outdated, and there will be follow up work that will define the CSP using modern MRT features.